spec/support/controller_macros.rb

```diff
@@ -1,4 +1,3 @@
 # frozen_string_literal: true
 
 module ControllerMacros; end
```

Comment:
this had to be done in the last commit.
Please read about `git commit --amend` option